test/android/core/javatests/src/org/xwalk/core/xwview/test/SettingsTest.java

```diff
@@ -17,10 +19,80 @@
     private static final String USER_AGENT =
             "Chrome/44.0.2403.81 Crosswalk/15.44.376.0 Mobile Safari/537.36";
     private static final String LANGUAGE = "jp";
+    private TestWebServer mWebServer;
 
     @Override
     public void setUp() throws Exception {
         super.setUp();
+        mWebServer = TestWebServer.start();
+    }
+
+    @Override
+    public void tearDown() throws Exception {
+        mWebServer.shutdown();
+        super.tearDown();
+    }
+
+    @MediumTest
+    @Feature({"Settings"})
+    public void testCacheMode() throws Throwable {
+        final String htmlPath = "/testCacheMode.html";
```

Comment:
All the the testing files such as "testCacheMode.html" is non-existent, it has effect because the set up of `mWebServer.setResponse(htmlPath, "response", null)`. Actually I also need study the Chromium test framework, can't explain more clear.